config/dist/skr/crd/bases/providers/gcp/cloud-resources.kyma-project.io_ipranges_ui.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+data:
+  details: |-
+    body:
+        - widget: Labels
+          source: metadata.name
+          name: metadata.name
+        - widget: Labels
+          source: spec.cidr
+          name: spec.cidr
+        - widget: Labels
+          source: status.cidr
+          name: status.cidr
+        - widget: Labels
+          source: status.state
+          name: status.state
+  form: |-
+    - path: spec.cidr
+      name: spec.cidr
+      widget: Text
+      simple: true
+      required: true
+  general: |
+    resource:
+        kind: IpRange
+        group: cloud-resources.kyma-project.io
+        version: v1beta1
+    urlPath: ipranges
+    name: Ip Ranges
+    scope: cluster
+    category: Discovery and Network
+    icon: tnt/network
+    description: >-
+        IpRange description here
+  list: |-
+    - source: spec.cidr
+      name: spec.cidr
+      sort: true
+    - source: metadata.namespace
+      name: metadata.namespace
+      sort: true
```

Comment:
Is the `metadata.namespace` field available on IpRange?

Reply:
Yes. Currently, the IpRange CRD is scope is Namespaced. See the example iprange, `iprange-sample-nikola` on the gardener cluster:

```
apiVersion: cloud-resources.kyma-project.io/v1beta1
kind: IpRange
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"cloud-resources.kyma-project.io/v1beta1","kind":"IpRange","metadata":{"annotations":{},"name":"iprange-sample-nikola","namespace":"kyma-dev"},"spec":{"cidr":"10.21.30.0/24"}}
  creationTimestamp: "2024-05-10T09:06:41Z"
  finalizers:
  - cloud-resources.kyma-project.io/deletion-hook
  generation: 1
  labels:
    cloud-resources.kyma-project.io/id: dd488e4f-6eea-4b70-8309-3400ad6ca762
  name: iprange-sample-nikola
  namespace: kyma-dev
  resourceVersion: "68204427"
  uid: fd877864-5550-4d78-8667-d34d29646deb
spec:
  cidr: 10.21.30.0/24
status:
  cidr: 10.21.30.0/24
  conditions:
  - lastTransitionTime: "2024-05-21T17:48:40Z"
    message: IpRange provisioned in GCP.
    reason: Ready
    status: "True"
    type: Ready
  id: dd488e4f-6eea-4b70-8309-3400ad6ca762
  state: Ready
```